### ISSUE TITLE
Is this abandonware?

### ISSUE THREAD

**Issue by daemontools** [jaybaird/python-bloomfilter#25]

**Comment by nirizr** [comment]
Looks like [this](https://github.com/joseph-fox/python-bloomfilter) fork is trying to reanimate this module

**Comment by joseph-fox** [comment]
@daemontools get the latest one from https://pypi.python.org/pypi/pybloom_live/  .

**Comment by indera** [comment]
@ joseph-fox  can you save the filter bits and reload them from a file?

**Comment by joseph-fox** [comment]
@indera  It seems to me [pickle](https://docs.python.org/2/library/pickle.html) is what you are looking for if the data security is not your primary concern.

**Comment by indera** [comment]
@joseph-fox I was looking for `bt.tofile()` and `BloomFilter.fromfile()` 

with open('bloom.bits', 'wb') as fw:
    bf.tofile(fw)

with open('bloom.bits', 'rb') as fr:
    bf2 = BloomFilter.fromfile(fr)

**Comment by joseph-fox** [comment]
@indera Yes, these are totally working but please use `ScalableBloomFilter`.
